mochify/lib/validate-config.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+exports.validateConfig = validateConfig;
+
+function validateConfig(config) {
```

Comment:
I left this function intentionally simple (i.e. manual checking) for now. We can start building our declarative mini DSL later ...